packages/babel-plugin-transform-block-scoping/package.json

```diff
@@ -14,7 +14,8 @@
   "main": "lib/index.js",
   "dependencies": {
     "@babel/helper-plugin-utils": "^7.10.4",
-    "lodash": "^4.17.13"
+    "lodash.assignin": "^4.2.0",
+    "lodash.values": "^4.3.0"
```

Comment:
Can we use `Object.values()` and get rid of this dep?

Reply:
NB: I attempted this in #11790, but it seems that `Object.values` is only fully supported from Node 7.x onwards.